### ISSUE TITLE
IATI CoVE template - xpath doesn't include iati-activities/iati-activity/

### ISSUE THREAD

**Issue by stevieflow** [OpenDataServices/cove#732]
I understand we don't cite the "full" path in the template.  Thinking ahead, do we therefore lose the ability to transform org file data, which has the exact same elements in some cases (``document-link``) , but prefixed with an ``iati-organisations/iati-organisation/`` path.

Do we tolerate this?  if this OK?

Just checking - based on a chat with @rory09

**Comment by Bjwebb** [comment]
You're right basically.

If we really wanted to support org files, we could have a magic field in the Meta tab.

**Comment by stevieflow** [comment]
Is the alternative to have the full path>?

**Comment by Bjwebb** [comment]
Yes. It would break compatibility with what we have now, and make IATI vs OCDS oddly inconsistent.

**Comment by stevieflow** [comment]
@bjwebb and I discussed this.  Whilst we know this is a break from other CoVE instances, it might well be worth doing this as part of the next sprint, as it is foundations we need

Happy to be pursued otherwise --> @timgdavies @robredpath

**Comment by robredpath** [comment]
Does this make the title row stuff more important as well, @Bjwebb ? Or is it unrelated?

**Comment by Bjwebb** [comment]
> Does this make the title row stuff more important as well, @Bjwebb ? Or is it unrelated?

Yes, in that the xpath headings will be harder for a human to tell apart, so a title row is more useful.

**Comment by stevieflow** [comment]
Have we a specific issue for IATI titles btw?

**Comment by Bjwebb** [comment]
It's not IATI specific, but I think we want to take this approach: https://github.com/OpenDataServices/flatten-tool/issues/130

**Comment by stevieflow** [comment]
I'm not sure we got to a decision on this - any idea @Bjwebb @robredpath ?

**Comment by Bjwebb** [comment]
Not properly made a decision, but we've been implicitly sticking with the status quo. In terms of implementation, I think it's easier to stick with the short path, but give some option to override the top level elements in the file (ie. using options, as for the number of rows to skip).

**Comment by stevieflow** [comment]
@Bjwebb @robredpath I think this is worth a review.  I note that the Meta tab is where we store the attributes for the root element (iati-activities / iati-organisations) , which is slightly counter-intuiative

Elsewhere, in terms of the controls that can be set on Metatab, are there any developments to isolated these away from data for the standard?
